scripts/comsos_multinode.sh

```diff
@@ -0,0 +1,274 @@
+done
+
+echo "--------Start initializing the chain ($CHAINID)---------"
+
+for (( a=1; a<=$NODES; a++ ))
+do
+    echo "-------Init chain ${a}--------"
+    echo "Deamon home :: $DAEMON_HOME-${a}"
+    $DAEMON init --chain-id $CHAINID $DAEMON_HOME-${a} --home $DAEMON_HOME-${a}
+done
+
+# add validators
+echo "---------Creating $NODES keys-------------"
+
+for (( a=1; a<=$NODES; a++ ))
+do
+    $DAEMON keys add "validator${a}" --keyring-backend test --home $DAEMON_HOME-${a}
+done
+
+# add accounts if any arguments passed
+if [ -z $ACCOUNTS ] || [ "$ACCOUNTS" -eq 0 ]
+then
+    echo "----- Accounts argument didn't pass, so not creating any accounts --------"
+else
+    echo "---------Creating $ACCOUNTS accounts-------------"
+
+    for (( a=1; a<=$ACCOUNTS; a++ ))
+    do
+        $DAEMON keys add "account${a}" --keyring-backend test --home $DAEMON_HOME-1
+    done
+fi
+
+echo "----------Genesis creation---------"
+
+for (( a=1; a<=$NODES; a++ ))
+do
+    if [ $a == 1 ]
+    then
+        $DAEMON --home $DAEMON_HOME-$a add-genesis-account validator$a 1000000000000$DENOM  --keyring-backend test
+        echo "done $DAEMON_HOME-$a genesis creation "
+        continue
+    fi
+    $DAEMON --home $DAEMON_HOME-$a add-genesis-account validator$a 1000000000000$DENOM  --keyring-backend test
+    $DAEMON --home $DAEMON_HOME-1 add-genesis-account $($DAEMON keys show validator$a -a --home $DAEMON_HOME-$a --keyring-backend test) 1000000000000$DENOM
+done
+
+echo "----------Genesis creation for accounts---------"
+for (( a=1; a<=$ACCOUNTS; a++ ))
+do
+     # add accounts
+    echo "cmd ::$DAEMON --home $DAEMON_HOME-1 add-genesis-account $($DAEMON keys show account$a -a --home $DAEMON_HOME-1 --keyring-backend test) 1000000000000$DENOM"
+
+    $DAEMON --home $DAEMON_HOME-1 add-genesis-account $($DAEMON keys show account$a -a --home $DAEMON_HOME-1 --keyring-backend test) 1000000000000$DENOM
+done
+
+echo "--------Gentx--------"
+
+for (( a=1; a<=$NODES; a++ ))
+do
+    $DAEMON gentx validator$a 90000000000$DENOM --chain-id $CHAINID  --keyring-backend test --home $DAEMON_HOME-$a
+done
+
+echo "---------Copy all node genesis to $DAEMON_HOME-1----------"
+
+for (( a=2; a<=$NODES; a++ ))
+do
+    cp $DAEMON_HOME-$a/config/gentx/*.json $DAEMON_HOME-1/config/gentx/
+done
+
+echo "----------collect-gentxs------------"
+
+$DAEMON collect-gentxs --home $DAEMON_HOME-1
+
+echo "---------Updating $DAEMON_HOME-1 genesis.json ------------"
+
+sed -i "s/172800000000000/600000000000/g" $DAEMON_HOME-1/config/genesis.json
+sed -i "s/172800s/600s/g" $DAEMON_HOME-1/config/genesis.json
+sed -i "s/stake/$DENOM/g" $DAEMON_HOME-1/config/genesis.json
+
+echo "---------Distribute genesis.json of $DAEMON_HOME-1 to remaining nodes-------"
+
+for (( a=2; a<=$NODES; a++ ))
+do
+    cp $DAEMON_HOME-1/config/genesis.json $DAEMON_HOME-$a/config/
+done
+
+echo "---------Getting public IP address-----------"
+
+IP="$(dig +short myip.opendns.com @resolver1.opendns.com)"
```

Comment:
If for some reason this command does not work we should take `127.0.0.1` as default

Reply:
okay.